spec/Records.tex

```diff
@@ -508,9 +420,9 @@ \section{The {\em this} Method}
 \section{The {\em these} Method}
 \index{records!iterating}
 
-A \chpl{these} method can be defined for records as well as classes (\rsec{The_these_Method}).  It
-provides an iterator which iterates over the contents of the record in a
-user-defined manner.
+A \chpl{these} method can be defined for records as well as classes
+(\rsec{The_these_Method}).  It provides an iterator which iterates over
```

Comment:
Correction: let's replace the entire sentence with:

```
It provides the iterator to be used when iterating over
the record in a \chpl{for} loop.
```

Separately, in the previous sentence, should we clarify for the "these" method to be "without arguments" ?